site/docs/components/App.tsx

```diff
@@ -27,20 +25,51 @@ const wagmiConfig = createConfig({
 });
 
 export default function App({ children }: { children: ReactNode }) {
+  const [theme, setTheme] = useState('light');
+
+  useEffect(() => {
```

Comment:
With the breaking 0.34.0 update, we need to adjust how dark and light mode is set. This updates our components to update with the appropriate theme.

Reply:
I can add this is a separate PR if preferred.